package.json

```
@@ -11,10 +11,9 @@
     "mp3"
   ],
   "author": "Tim Koschuetzki <tim@transloadit.com>",
-  "packageManager": "yarn@1.22.19+sha1.4ba7fc5c6e704fce2066ecbfb0b0d8976fe62447",
+  "packageManager": "yarn@4.0.1",
   "engines": {
-    "node": ">= 14.16",
-    "yarn": ">= 1.22.19"
+    "node": ">= 18"
```

We should revert this. The `engines` field tells the engines users should use. So this should _never_ contain any package managers, and the `node` engine should be restored to its original value.

```suggestion
    "node": ">= 14.16",
```

---

you mean revert the node change but not revert the yarn change?

---

Yes